src/index.html

```diff
@@ -1,13 +1,338 @@
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="products__description">
+              Audio sunglasses
+            </div>
+          </div>
+
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--5-8 grid__item--tablet--2-5">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+      <section class="page__section" id="about-us">
+        <div class="container">
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--1-5 grid__item--tablet--1-3">
+              <h2 class="page__section-title">
+                Why buy direct from Bose
+              </h2>
+            </div>
+            <div class="grid__item grid__item--6-10 grid__item--tablet--4-6">
+              <h3 class="page__section-subtitle">Free 2-day shipping and returns</h3>
+              <h3 class="page__section-subtitle">90-day risk-free trial</h3>
+              <h3 class="page__section-subtitle">World class customer service</h3>
+              <h3 class="page__section-subtitle">My Bose account management</h3>
+              <p>
+                A great product is more than what’s in 
+                the box. It’s a promise of premium performance,
+                world-class support, and everything you expect
+                from a trusted brand. It’s just one of many
+                reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--1-5 grid__item--tablet--1-3">
+              <form action="#" method="get">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__form-field"
+                  required
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contact-us__form-field"
+                  required
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  minlength="1"
+                  class="contact-us__form-field contact-us__form-field--textarea"
+                  required
+                ></textarea>
+
+                <button class="button contact-us__button">
+                  Send
+                </button>
+              </form>
+            </div>
+            <div class="grid__item grid__item--8-10 grid__item--tablet--4-6">
```

Comment:
I guess, it should be more space between button and phone-subtitle on the phone version
https://prnt.sc/KYQOUklMAGWW